cfglib.py

```diff
@@ -111,6 +111,8 @@
                 help="Migrate all VM's from all tenants. User, specified in "
                      "the 'dst' section of config also should have admin role "
                      "in all tenants."),
+    cfg.BoolOpt('all_images', default=False,
```

Comment:
We'd have to define some better way of working with subsets of resources in future, there's way too many options `all_.*`. IMHO there should only be 1 option: tenant to be migrated, CF would then have to automatically determine all the resources specified tenant has access to and migrated those. That's just a note for future, no changes required.